src/plugins/intel_gpu/tests/functional/single_layer_tests/gather_elements_padding_input.cpp

```diff
@@ -83,6 +83,26 @@ class GatherElementsPaddingInputTest : public testing::WithParamInterface<Gather
         inType = outType = input_precision;
         function = init_subgraph(input_params, input_precision);
     }
+
+    void generate_inputs(const std::vector<ov::Shape>& targetInputStaticShapes) override {
+        inputs.clear();
+        const auto& funcInputs = function->inputs();
+        for (size_t i = 0lu; i < funcInputs.size(); i++) {
+            const auto& funcInput = funcInputs[i];
+            ov::Tensor tensor;
+            if (funcInput.get_element_type().is_real()) {
+                ov::test::utils::InputGenerateData in_data;
+                in_data.start_from = 0;
+                in_data.range = 10;
```

Comment:
I think it would be better to use original dimension size here, can it be obtained from GatherElements input?